internal/pkg/rpaas/k8s.go

```diff
@@ -1289,6 +1289,11 @@ func (m *k8sRpaasManager) validateFlavors(ctx context.Context, instance *v1alpha
 		if flavorObj.Spec.CreationOnly && !isCreation {
 			return &ValidationError{Msg: fmt.Sprintf("flavor %q can used only in the creation of instance", f)}
 		}
+
+		incompatibleFlavor := checkIncompatibleFlavors(flavorObj.Spec.IncompatibleFlavors, flavors)
```

Comment:
I think is just cosmestic, but would be nice if `checkIncompatibleFlavors` returns a error instead a string to be more golang way to handle error